docs/integrations/microsoft-azure/opentelemetry/sql-server-opentelemetry.md

```diff
@@ -0,0 +1,177 @@
+
+### Step 1: Set up Collector
+
+If you want to use an existing Otel Collector then this step can be skipped by selecting the option of using an existing Collector.
+
+If you want to create a new Collector please select **Add a new Collector** option.
+
+Select the platform for which you want to install the Sumo OpenTelemetry Collector.
+
+This will generate a command which can be executed in the machine which needs to get monitored. Once executed it will install the Sumo Logic OpenTelemetry Collector agent.
+
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/SQLServer-OpenTelemetry/SQL-Server-Collector.png')} alt="Collector" />
+
+### Step 2: Configure integration
+
+The Microsoft SQL Server App's queries and dashboards depend on logs from the SQL Server ERRORLOG, which is typically found in:
+
+`C:\Program Files\Microsoft SQL Server\MSSQL<version>.MSSQLSERVER\MSSQL\Log\ERRORLOG*`
+
+You can add any custom fields which you want to tag along with the data ingested in sumo.
+Click on the **Download YAML File** button to get the yaml file.
+
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/SQLServer-OpenTelemetry/SQL-Server-YAML.png')} alt="YAML" />
+
+### Step 3: Sending logs to Sumo
+
+Once you have the yaml file downloaded in step 2, you can copy it to the machine which needs to be monitored. Follow the below steps based on the platform of the machine:
+
+<Tabs
+  className="unique-tabs"
+  defaultValue="Linux"
+  values={[
+    {label: 'Linux', value: 'Linux'},
+    {label: 'Windows', value: 'Windows'},
+    {label: 'macOS', value: 'macOS'},
+  ]}>
+
+<TabItem value="Linux">
+
+1.  Copy the yaml file to `/etc/otelcol-sumo/conf.d/` folder in the SQL Server instance which needs to be monitored.
+2.  restart the collector using
+  ```sh
+  sudo systemctl restart otelcol-sumo
+  ```
+
+</TabItem>
+<TabItem value="Windows">
+
+1.  Copy the yaml file to `C:\ProgramData\Sumo Logic\OpenTelemetry Collector\config\conf.d` folder in the machine which needs to be monitored.
+2.  Restart the collector using 
+  ```sh
+  Restart-Service -Name OtelcolSumo
+  ```
+
+</TabItem>
+<TabItem value="macOS">
+
+1.  Copy the yaml file to `/etc/otelcol-sumo/conf.d/` folder in the SQL Server instance which needs to be monitored.
+2.  Restart the otelcol-sumo process using the below command 
+  ```sh
+  otelcol-sumo --config /etc/otelcol-sumo/sumologic.yaml --config "glob:/etc/otelcol-sumo/conf.d/*.yaml"
+  ```
+
+</TabItem>
+</Tabs>
+
+After successful execution of the above command, Sumo will start receiving the data from your host machine.
+
+Press **Next** . This will install the app to your Sumo Logic Org. The app consists of Dashboards.
+
+Panels will start to fill automatically. It's important to note that each panelfills with data matching the time range query and received since the panel was created. Results won't immediately be available, but within 20 minutes, you'll see full graphs and maps.
```

Comment:
```suggestion
Panels will start to fill automatically. It's important to note that each panel fills with data matching the time range query and received since the panel was created. Results won't immediately be available, but within 20 minutes, you'll see full graphs and maps.
```